content/version/2/0/code_of_conduct.de.md

```diff
@@ -0,0 +1,137 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmende und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für alle, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Erfahrungsstufe, Bildung, sozialem Status, Nationalität, persönlicher
+Erscheinung, Rasse, Religion oder Geschlechtsidentität und sexueller
+Orientierung.
+
+Wir verpflichten uns, in einer Weise zu handeln und zu interagieren, die zu
+einer offenen, einladenden, vielfältigen, integrativen und gesunden Gemeinschaft
+beiträgt.
+
+## Unsere Standards
+
+Beispiele für Verhaltensweisen, die zu einem positiven Umfeld für unsere
+Gemeinschaft beitragen, sind:
+
+- Einfühlungsvermögen und Freundlichkeit gegenüber anderen Menschen zeigen
+- Unterschiedliche Meinungen, Standpunkte und Erfahrungen respektieren
+- Konstruktives Feedback geben und würdevoll annehmen
+- Verantwortung übernehmen und uns bei denjenigen entschuldigen, die von unseren
+  Fehlern betroffen sind, und aus den Erfahrungen lernen
+- Konzentration auf das, was nicht nur für uns als Individuen, sondern für die
+  gesamte Gemeinschaft das Beste ist
+
+Beispiele für inakzeptables Verhalten sind:
+
+- Die Verwendung sexualisierter Sprache, Bilder oder Symbolik sowie unerwünschte
+  Versuche sexueller Anbahnung
+- Beleidigende oder abwertende Kommentare, persönliche oder politische Angriffe,
+  Nutzung des Internets für unbilliges Verhalten
```

Comment:
What is `Nutzung des Internets für unbilliges Verhalten` referring to trolling? I think there is no translation for trolling

Reply:
This point has not changed from 1.4 to 2.0 (except that the `/` becomes an `or`). So I left it as it was in the previous version.

I'm open to suggestions on how the word `trolling` can be integrated when needed.

Reply:
how about "und trollen"
https://de.wikipedia.org/wiki/Troll_(Netzkultur)

Reply:
I added it. Thank You.